creating_packages/toolchains/cmake.rst

```diff
@@ -8,13 +8,20 @@ CMakeToolchain
 
     This is an **experimental** feature subject to breaking changes in future releases.
 
+.. warning:
 
-The ``CMakeToolchain`` can be used in the ``toolchain()`` method:
+    Starting in Conan 1.32 ``write_toolchain_files()`` method and ``toolchain`` attribute have been
+    deprecated and moved. They will be removed in Conan 1.33, please use ``generate()`` instead of
+    ``write_toolchain_files()`` and ``generate`` or ``generators = "ToolChainClassName"`` instead of
+    ``toolchain`` attribute instead.
```

Comment:
Here in the classes it makes sense to talk about the `write_toolchain_files` method of toolchain classes, probably about the `toolchain()` one of ConanFile too. But this paragraph needs some rewording.